packages/docs/docs/getting-started/quickstart.mdx

```diff
@@ -28,14 +30,21 @@ node -v
 
 ### Creating a new project
 
+Navigate to whichever directory you would like your new animation folder 
+to be placed.  
+
 Run the following command in order to scaffold a new Motion Canvas project:
 
 ```bash
 npm init @motion-canvas
 ```
 
-Answer the prompts to name your project and select which language you would like
-to use; either TypeScript or plain JavaScript.
+Enter the name of your project (Ex] my-first-animation)
+
+Enter the project path (Ex] my-first-animation)
```

Comment:
"Ex]" is a really peculiar way of denoting an example, can we change it to:
```suggestion
Enter the project path (e.g. `my-first-animation`)
```
?
Putting text around the ticks (`` ` ``) will make it look console-like, `like this`.

Reply:
Support this. The first time I read this I was a bit confused about this notation.